Vizzuality/heco-invest
# refactoring: PD uses Mosaic Enum instead of Region Locations

We decided to use Mosaic Enum types during Project developer creation, therefore this PR is adding such Enum type and also removing `has_many :locations` relation from Project developer model (this one is no longer needed, because it was fully replaced by Mosaic Enum).

## Testing instructions

There are automatic tests which ensure that new Mosaic attribute is properly setup for PD records. Everything can be also verified via Rswag documentation ;)

## Tracking

https://vizzuality.atlassian.net/browse/LET-314
